pkg/client/interface.go

```diff
@@ -46,6 +46,7 @@ type SupplierClient interface {
 	// smt.SparseMerkleClosestProof, corresponding to some previously created claim
 	// for the same session. The proof is validated on-chain as part of the pocket
 	// protocol.
+	// TODO_IMPROVE(@h5law): Use a SparseCompactClosestProof here instead.
```

Comment:
#PUC on why.

Thoughts on just creating a ticket that says "migrate to compact proofs on-chain"?

That way:
1.  You can reference the same ticket in multiple places
2. Make a deliverable be "search for all #XXX in the codebase and resolve it"

Reply:
https://github.com/pokt-network/poktroll/issues/427